README.md

```diff
@@ -262,6 +262,21 @@ String result = metadata.get("Result");
 
 Mainly for debug purposes, jchess provides a simple API to print a position in CLI:
 
+#### Unicode
+
+The unicode renderer's output is compact but does not support showing which cells are dark or light when they are not empty.
+
+```java
+JChessGame game = JChessGame.newGame();
+UnicodePositionRenderer.render(System.out, game.getPosition());
+```
+
+![image](https://user-images.githubusercontent.com/18663098/159409079-d4f994d3-2051-404a-aec7-951e0a4a7755.png)
+
+Two extra params can be passed to `UnicodePositionRenderer.render` to customize the empty cells display String
```

Comment:
A suggested above, isn't it better to simply remove the rendering of an empty black cell ? :thinking: I think it does not really improve the "readability" of the position...

This comment should therefore by removed.